common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -833,6 +834,48 @@ public static int hashCodeAscii(CharSequence bytes) {
         return null;
     }
 
+    private static class Mpsc {
+        private static boolean USE_MPSC_CHUNKED_ARRAY_QUEUE;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "USE_MPSC_CHUNKED_ARRAY_QUEUE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS3008)